### PR TITLE
Issue 5239 simulated relationship behavior fixes

### PR DESCRIPTION
**Pull request by psikomonkie**
Simulated Relationships allowed females to get pregnant and give birth in the same week, letting them have more kids than possible. Characters in simulated relationships with females were unable to procreate, and if they did their children wouldn't be saved. Children from simulated relationships will have a high school education if they're at least 16.

Solves issue #5239

### CODE REVIEW

**Comment by psikomonkie** [comment]
Hey @IllianiCBT , this is ready for review. I don't think I can add assignees, so a comment it is. Thanks!

**Comment by IllianiCBT** [comment]
Perfect. I'm out of the office atm, but I'll take a butchers once I'm done.

Welcome to the project! :)

File: MekHQ/docs/history.txt [hunk 1]
```diff
@@ -21,6 +21,7 @@ MEKHQ VERSION HISTORY:
 + PR #5228: Added Monthly StratCon Support Points Generation #5228
 + Fix #5181: Generate required lances for manual AtB contracts for StratCon init
 + PR #5234: Refined Turnover Prompt Dialog Text
++ Fix #5239: Fixed Simulated Relationship Behavior
```
Comment (IllianiCBT) [inline MekHQ/docs/history.txt:24]: So this is something we handle _when_ the PR is merged. So it shouldn't be done as part of the PR. When we do a bulk merge, having multiple PRs affecting single lines in history.txt will just create a pile of conflicts and will make everyone sad.

File: MekHQ/src/mekhq/campaign/personnel/procreation/AbstractProcreation.java [hunk 2]
```diff
@@ -466,7 +466,12 @@ public List<Person> birthHistoric(final Campaign campaign, final LocalDate today
             baby.setLoyalty(Compute.d6(3) + 2);
 
             // set education based on age
-            baby.setEduHighestEducation(EducationLevel.EARLY_CHILDHOOD);
+            if(baby.getAge(today) < 16) {
+                baby.setEduHighestEducation(EducationLevel.EARLY_CHILDHOOD);
+            }
```
Comment (IllianiCBT) [inline MekHQ/src/mekhq/campaign/personnel/procreation/AbstractProcreation.java:471]: Very nit-picky, but can the `else` be on the same line as the `}` as that's how we format it across the codebase

**Review by IllianiCBT — CHANGES_REQUESTED** [review]
A couple of very small changes, otherwise looks good :)

File: MekHQ/docs/history.txt (hunk 1 above)
Comment (psikomonkie) [reply MekHQ/docs/history.txt]: That took a few tries to resolve properly, but it's fixed I think.

File: MekHQ/src/mekhq/campaign/personnel/procreation/AbstractProcreation.java (hunk 2 above)
Comment (psikomonkie) [reply MekHQ/src/mekhq/campaign/personnel/procreation/AbstractProcreation.java]: Yep, done, sorry I missed that it's even in the style guide.

**Review by IllianiCBT — APPROVED** [review]
Sorry for the slower reply, I'm having to play host for Thanksgiving, so my computer time is sporadic.

**Pull request merged**